public/app/features/alerting/components/DeprecationNotice.tsx

```diff
@@ -5,23 +5,19 @@ import { Alert } from '@grafana/ui';
 export const LOCAL_STORAGE_KEY = 'grafana.legacyalerting.unifiedalertingpromo';
 
 const DeprecationNotice = () => (
-  <Alert severity="warning" title="Grafana legacy alerting is going away soon">
+  <Alert severity="warning" title="Grafana legacy alerting is deprecated and will be deleted soon">
     <p>
-      You are using Grafana legacy alerting, it has been deprecated and will be removed in the next major version of
-      Grafana.
+      You are using Grafana legacy alerting which is deprecated since Grafana 9.0. Legacy alerting will be removed from
+      Grafana in an upcoming release without further notice.
```

Comment:
```suggestion
      You are using Grafana legacy alerting, which has been deprecated since Grafana 9.0. It will be removed in a future release.

```